packages/core/src/lib/androidSdk/AndroidSdkTools.ts

```diff
@@ -32,6 +33,19 @@ export class AndroidSdkTools {
   private jdkHelper: JdkHelper;
   private pathJoin: (...paths: string[]) => string;
 
+  static async newAndroidSdkTools(process: NodeJS.Process, config: Config, jdkHelper: JdkHelper,
+      log: Log = new ConsoleLog('AndroidSdkTools')): Promise<AndroidSdkTools> {
+    // eslint-disable-next-line @typescript-eslint/no-use-before-define
+    if ((await validatePath(config)).isError()) {
+      throw new Error(`androidSdkPath does not exist: ${config.androidSdkPath}`);
+    }
```

Comment:
This entire block can be replaced with the following:

```suggestion
    (await AndroidSdkTools.validatePath(config)).unwrap();
```

Also, `eslint-disable-next-line` is a last resort. Let's avoid using it.

Reply:
If there is an error it will throw it by itself?
And about the eslint-disable-next-line, it tells me that I use the function before I define it.. the only way that I can define it before I'm using it is to remove it from the class and write it at the start of the file, but we wanted it to be static..

Reply:
Check the [`Result.unwrap()`](https://github.com/GoogleChromeLabs/bubblewrap/blob/master/packages/core/src/lib/Result.ts#L74-L79) implementation. If you invoke `validatePath` with `AndroidSdkTools.validatePath()`, does it complain?